### ISSUE TITLE
unused import false positive default Laravel User class

### ISSUE THREAD

**Issue by loganhenson** [tightenco/tlint#52]

**Comment by sbine** [comment]
I think this is caused by the same bug as #49, right?

```
Lints for app/User.php
============
! There should be no unused imports.
6 : `use Illuminate\Foundation\Auth\User as Authenticatable;`
```

**Comment by loganhenson** [comment]
More than likely
